components/dashboard/src/settings/Preferences.tsx

```diff
@@ -98,6 +98,17 @@ export default function Preferences() {
             </div>
             {useDesktopIde && <>
                 <div className="mt-4 space-x-4 flex">
+                    <SelectableCard className="w-36 h-40" title="VS Code Desktop" selected={defaultDesktopIde === 'code-desktop'} onClick={() => actuallySetDefaultDesktopIde('code-desktop')}>
```

Comment:
Should we just use `VC Code`?

It does not feel in the screen and it is already under desktop IDEs section?